ingestion/src/metadata/ingestion/source/dashboard/tableau/metadata.py

```diff
@@ -268,6 +270,46 @@ def yield_dashboard(
                 )
             )
 
+    @staticmethod
+    def _get_data_model_column_fqn(
+        data_model_entity: DashboardDataModel, column: str
+    ) -> Optional[str]:
+        """
+        Get fqn of column if exist in table entity
+        """
+        if not data_model_entity:
+            return None
+        for tbl_column in data_model_entity.columns:
+            for child_column in tbl_column.children or []:
+                if column.lower() == child_column.name.__root__.lower():
+                    return child_column.fullyQualifiedName.__root__
+        return None
+
+    def _get_column_lineage(
```

Comment:
We should add try/except to these methods so that if something fails atleast the normal lineage should get computed